internal/sdkv2provider/resource_cloudflare_access_group_test.go

```diff
@@ -613,6 +638,18 @@ resource "cloudflare_access_group" "%[1]s" {
 }`, resourceName, identifier.Type, identifier.Identifier)
 }
 
+func testAccCloudflareAccessGroupConfigEmailList(resourceName string, identifier *cloudflare.ResourceContainer) string {
+	return fmt.Sprintf(`
+resource "cloudflare_access_group" "%[1]s" {
+  %[2]s_id = "%[3]s"
+  name     = "%[1]s"
+
+  require {
+		email_list = ["e3a0f205-c525-4e48-a293-ba5d1f00e638", "5d54cd30-ce52-46e4-9a46-a47887e1a167"]
```

Comment:
you'll need to create these as part of your test assertion here otherwise they will be invalid.

Reply:
How does the hardcoded ip_lists work in other tests? Aren't both of those the same teams_list resource, but just a different type? ([ref](https://github.com/cloudflare/terraform-provider-cloudflare/blob/8885d6ce2fccfb875a6bbdaa9508a4ecc1ee344b/internal/sdkv2provider/resource_cloudflare_access_group_test.go#L467))

Reply:
the IP lists are statically managed because they have dependencies across the plugin framework and SDKv2 (cyclic import workaround). as email list is new and doesn't have the same constraints, you need to add it as part of this assertion.

Reply:
Added the resource in acceptance tests